app/src/main/kotlin/me/myshows/android/api/jsonrpc/JsonRPCObjects.kt

```diff
@@ -0,0 +1,16 @@
+package me.myshows.android.api.jsonrpc
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class JsonRPCRequestObject<out T>(
+        @JsonProperty("method") val method: String,
+        @JsonProperty("param") val param: T,
+        @JsonProperty("jsonrpc") val jsonrpc: String = "2.0",
+        @JsonProperty("id") val id: Int = 1
+)
+
+data class JsonRPCResponseObject<out T>(
```

Comment:
I'll see later if it's possible to get error with 2xx code. If it's possible then we should serialize it (but I think into separate class), otherwise retrofit won't call converter in error case at all.

Reply:
Yes, if our request is incorrect we will get correct error object as response, so we should support this case as well